Algebraic Topology/algebraic-topology.tex

```diff
@@ -431,17 +436,202 @@ \section{Functoriality of $\fund$}
+  the tunnels).
+
+  $B'$ = $K \cap \{(x,y,z) \in \R^3 \mid z < \epsilon\}$ ($K$ minus
+  the bridges).
+
+  $A'$ is $m$ open line segments.  In our example, using $K =$
+  trefoil, $m = 3$.
+
+  Let $A = (A')^C$.  Then $\fund(A) \isom F_m$, the free group on a
+  set of size $m$.
+
+  Let $B = (B')^C$.  Then $\fund(B) \isom F_m$.
+
+  So $A \cap B = \R^3 \minus (A' \union B')$.
+  $\fund(A \cap B) \isom F_{2m}$
 
+  ((do we invoke Van Kampbell's theorem here?))
 
+  \includegraphics[scale=0.2]{images/mountains.jpg}
 
+  By sliding $a$ under the bridge, we see that $a$ and $b$ are
+  conjugate via $c$!
 
+  \includegraphics[scale=0.2]{images/mountains-with-explanation.jpg}
 
+  Finally, to compute the fundamental group of $K^C$, we pick an
+  arbitrary basepoint $x$ and put loops around the knot as follows.
+
+  We take exactly one loop per open line interval in $A'$.  For each
+  crossing, we examine the three loops involved (which are called $a$,
+  $b$, and $c$ above) and determine the appropriate conjugate
+  relation.  The fundamental group is the group generated by the loops
+  and modded out by the relations.
+
+  In the case of the trefoil, the fundamental group
+  is $${F_m}/{(a=c*b*c^{-1}, b=a*c*a^{-1}, c=b*a*b^{-1})}$$, as
+  illustrated below.
+
+  \includegraphics[scale=0.2]{images/trefoil-fully-described.jpg}
+\end{example}
+\begin{thm}[Wirtinger]
+  For any knot $K$, $\fund(\R^3\minus K)$ is generated by loops
+  $\gamma_1,,,\gamma_m$ over crossings modulo $a = cbc^{-1}$.
+\end{thm}
+
+
+\section{CW complexes}
+
+CW complexes are like a space built by gluing together balls.
+\begin{defn}
+  A \de{CW complex} is a space $X$ which is a union of
+  $X_0 \subset X_1 \subset \subset \subset$ such that $X_n$ is
+  constructed from $X_{n-1}$ by taking
+  $X_{n-1} \dunion_{\alpha} D_{\alpha}^n$ modulo identifying (( --> he
+  might have meant $S^n$ here.  i think the point is that he meant
+  $\d D^n$)) $\phi_\alpha \colon S_\alpha^{n-1} \to X_{n-1}$.  $X_n$
+  is called $n$-Skeleton.  $X ~ \phi_\alpha$.
+
+  with topology s.t. $A$ is open iff $A \cap X_n$ is open for all $n$.
+\end{defn}
+\begin{example}
+  $S^n = D^{n-1}/\text{boundary}$ all goes to one point.  $X_0 =$
+  discrete set of points.  The $0$-skeleton is a point.  $X_1 =$
+  graph. (by connecting the points in $X_0$.)  $X_2 =$ take each cycle
+  (can go around multiple times) in $X_1$ and glue it to the boundary
+  of a disk (can go around multiple times).
+\end{example}
+\begin{example}
+  ((this example is incomplete and will be repeated next lecture))
+  $\R P^n =$ lines in $\R^{n+1}$.
+
+  $\R P^n = \{ \R \dot (1, a_1,,, a_{n-1})\} \isom \R^{n}$.
+  $\bunion \{ \R \dot (0, a_1,,, a_{n-1})\} \isom \R P^{n-1}$.
+\end{example}
+
+\mydate{d7}{6}{2}{2017}
+
+We wish to show how \(\R P^n\) is a \(CW\)-complex. First let the
+\(n-1\) skeleton be \(\R P^{n-1}\). We add the \(n\) cell \(D^n\) to
+structure under the gluing map \(S^{n} \to \R P^{n-1}\) where
```

Comment:
Maybe he meant S^{n-1}?  The picture that I do understand in my mind is the representation of RP^2 (lines in R3) as a CW complex.  It looks like a disk D2 with half the circumference identified with the other half of the circumference.

In that particular case, what would the glueing map be?  Would it be glueing the two circumference halves (S^1) to itself?